modules/swagger-parser/src/main/java/io/swagger/parser/SwaggerResolver.java

```diff
@@ -53,4 +63,26 @@ public Swagger resolve() {
 
         return swagger;
     }
+
+    public static class Settings {
```

Comment:
I see.  So the parser is essentially copying the parameters from the path-item object to the operations "below" it in hierarchy correct?  and what you're saying is, you don't want it to copy them, or you want to copy and leave them there?